src/utils/skipLogic.js

```diff
@@ -141,7 +141,25 @@ module.exports = function () {
     var checkEval = null
     for (var i = 0; i < check.group.length; i++) {
       var c = check.group[i]
-      var value = scope.form[c.variable].$modelValue
+      var fieldVariableSplit = c.variable.split('.')
+      var value
+      switch (fieldVariableSplit[0]) {
+        case 'form':
+          value = scope.form[fieldVariableSplit[1]][fieldVariableSplit[2]]
+          break
+        case 'globals':
+          var property = c.variable.replace('globals.', '')
+          var propertyArry = property.split('.')
```

Comment:
typo propertyArry.  It is also not intuitive what it is doing.  maybe create a function called walkPath and pass in the object and the path and return the value.

Reply:
great thanks @napiergit i have addressed those comments.